pkg/antctl/command_definition.go

```diff
@@ -236,6 +238,16 @@ func (cd *commandDefinition) validate() []error {
 	if len(cd.use) == 0 {
 		errs = append(errs, fmt.Errorf("the command does not have name"))
 	}
+	existingAliases := make(map[string]bool)
+	for _, a := range cd.aliases {
+		if a == cd.use {
+			errs = append(errs, fmt.Errorf("%s: command alias is the same with use of the command", cd.use))
+		}
+		if _, ok := existingAliases[a]; ok {
+			errs = append(errs, fmt.Errorf("%s: command alias is provided twice: %s", cd.use, a))
+		}
+		existingAliases[a] = true
+	}
```

Comment:
These could only be displayed if command definition validation fails, is that correct?

Reply:
You are right.